flowpipe/utilities.py

```diff
@@ -93,7 +113,7 @@ def get_hash(obj, hash_func=lambda x: sha256(x).hexdigest()):
     """
     try:
         return hash_func(obj)
-    except (TypeError, ValueError):
+    except TypeError:
```

Comment:
My datatypes (`np.ndarray`) sometimes raise a `ValueError` when you try to hexify them. Can we leave the `ValueError` or make the `try except` more generic?

Reply:
Sorry, that was my mistake, I did not intend to remove that, thanks for catching this